internal/services/cosmos/cosmosdb_postgresql_cluster_resource.go

```diff
@@ -636,3 +659,19 @@ func flattenMaintenanceWindow(input *clusters.MaintenanceWindow) []MaintenanceWi
 		},
 	}
 }
+
+func formatServerNames(input *[]clusters.ServerNameItem) []ServerNameItem {
```

Comment:
Minor but for consistency could we rename this to 
```suggestion
func flattenServerNames(input *[]clusters.ServerNameItem) []ServerNameItem {
```